org.scala-ide.sdt.core.tests/src/org/scalaide/core/testsetup/SDTTestUtils.scala

```diff
@@ -39,10 +39,11 @@ import org.eclipse.jdt.launching.JavaRuntime
 import org.scalaide.core.compiler.IScalaPresentationCompiler
 import org.scalaide.core.internal.project.ScalaProject
 
-/** Utility functions for setting up test projects.
+/**
+ * Utility functions for setting up test projects.
  *
  */
-object SDTTestUtils extends HasLogger {
+trait SDTTestUtils extends HasLogger {
```

Comment:
If there is something you don't like with the test suite, please change it and educate us. I just don't want to introduce new types everywhere without seeing a compelling reason.